0e240741724d
*** Found local files:
***   * /usr/local/airflow/logs/dag_id=process_web_log/run_id=manual__2023-11-24T16:51:29.691072+00:00/task_id=load_data/attempt=1.log
[2023-11-24, 16:51:30 UTC] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: process_web_log.load_data manual__2023-11-24T16:51:29.691072+00:00 [queued]>
[2023-11-24, 16:51:30 UTC] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: process_web_log.load_data manual__2023-11-24T16:51:29.691072+00:00 [queued]>
[2023-11-24, 16:51:30 UTC] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-24, 16:51:30 UTC] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_data> on 2023-11-24 16:51:29.691072+00:00
[2023-11-24, 16:51:30 UTC] {standard_task_runner.py:57} INFO - Started process 5362 to run task
[2023-11-24, 16:51:30 UTC] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'process_web_log', 'load_data', 'manual__2023-11-24T16:51:29.691072+00:00', '--job-id', '1465', '--raw', '--subdir', 'DAGS_FOLDER/process_web_log.py', '--cfg-path', '/tmp/tmpnwqa2wcb']
[2023-11-24, 16:51:30 UTC] {standard_task_runner.py:85} INFO - Job 1465: Subtask load_data
[2023-11-24, 16:51:30 UTC] {task_command.py:416} INFO - Running <TaskInstance: process_web_log.load_data manual__2023-11-24T16:51:29.691072+00:00 [running]> on host 0e240741724d
[2023-11-24, 16:51:30 UTC] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='process_web_log' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-24T16:51:29.691072+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-24T16:51:29.691072+00:00'
[2023-11-24, 16:51:30 UTC] {python.py:194} INFO - Done. Returned value was: None
[2023-11-24, 16:51:30 UTC] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=process_web_log, task_id=load_data, execution_date=20231124T165129, start_date=20231124T165130, end_date=20231124T165130
[2023-11-24, 16:51:30 UTC] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-24, 16:51:30 UTC] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check